pyproject.toml

```diff
@@ -61,6 +61,7 @@ uvloop = {version = "^0.20.0", optional = true}
 semver = "^3.0.2"
 llama-index = {version = ">=0.11.0", optional = true}
 boto3 = {version = ">1, <2", optional = true}
+h11 = "0.16.0"
```

Comment:
should range this right?

Reply:
or really, since this is a transient dependency, which core dependency do we need to increase the range of to allow users to install a safe version?